src/_msc/compare.py

```diff
@@ -0,0 +1,75 @@
+import argparse
+import torch
+import pickle
+import numpy as np
+import sys
+import yaml
+import subprocess
+import os
+
+sys.path.append('../')
+import pkgmgr as opentf
+
+with open('../__config__.yaml', 'r') as file:
+        config = yaml.safe_load(file)
+torch = opentf.install_import(config['data']['pytorch'], 'torch')
+
+def main():
+    parser = argparse.ArgumentParser(description="Compare two .pkl files (teamsvecs assumed)")
+    parser.add_argument("-a", required=True, help="Path to first .pkl file")
+    parser.add_argument("-b", required=True, help="Path to second .pkl file")
+    parser.add_argument("--sort", action="store_true", help="Sort rows by 'skill' key before comparison")
+    parser.add_argument("--print-shapes", action="store_true", help="Print detailed shape and content comparison using shape.py")
+    args = parser.parse_args()
+
+    def load_pkl(fp): return pickle.load(open(fp, 'rb'))
+
+    def to_dense_tensor(m, is_sorted_np_array=False):
+        if m is None: return None
+        if m.shape[0] == 0: return torch.empty((0, 1, m.shape[1] if m.ndim > 1 else 0), dtype=torch.uint8)
+        if is_sorted_np_array: return torch.from_numpy(m).type(torch.uint8)
+        return torch.tensor(m.toarray(), dtype=torch.uint8)
+
+    a, b = load_pkl(args.a), load_pkl(args.b)
+    eq = True
+    keys = ['skill', 'member', 'loc']
+
+    # Sort data first if requested
+    if args.sort:
```

Comment:
Hi @thangk 
we should not sort the arrays since it should be in the same order based on the year timestamps.

Reply:
ideally, changing the format of an array should not impact the array content like changing rows/columns/...